tests/staged/data/simple_cgw/CGW/staged.yaml

```diff
@@ -0,0 +1,50 @@
+header:
```

Comment:
There are a few issues with this:

First, just adding this file doesn't cause it to be used by any test.  You need to adjust or add at least one test which uses it.  The easiest way to do this is probably to copy an existing similar baseline case like tests/baseline/cases/staged-simple-comps.yml and tweak it to point at the simple_cgw directory.

The rest of the complaints I have are about the content of the file.  Now it is true that pushsource library doesn't parse this file in any way, so the content of this file doesn't really have to be correct.  You could put anything at all in here and the tests could still pass.  Right now though the content seems very misleading which makes me think there could be some misunderstandings.  I would rather that the file is either a correct CGW metadata file as will actually be used when doing a push to CGW, or else it's obviously just a placeholder for testing.

The problems are:
- file doesn't have to be named staged.yaml, this is different from the staged.yaml at root of the staging dir (which would be at tests/staged/data/simple_cgw/staged.yaml here - not in CGW directory).  It would be better to name it something else to reduce confusion.
- I don't think the file should have that "header", "payload" structure. It has to be in the top-level staged.yaml file purely for backwards-compatibility reasons.
- The file right now seems to be more like some kind of a schema than a valid CGW metadata file.  For example, the "state|present" string - obviously in reality it has to be one or the other.
- The payload right now has a list with probably more elements than was intended.  For example, right now it has:

```yaml
- type: file
- state: present|absent
- ...
```

So that's a list like: `[{"type": "file"}, {"state": "present|absent"}, ...]`.  But I suspect it was intended to instead write:

```yaml
- type: file
  state: present|absent
  ...
```

Which would give you something quite different: `[{"type": "file", "state": "present|absent", ...}, ...]`  In other words there are probably some '-' in the YAML which should not be there.

Do you have any *real* CGW metadata file which could be used for the test data here?  If not, I'd prefer it if this entire file were just a placeholder with a comment, like:

```yaml
# CGW metadata should go here.
# This library does not parse the file, and this is just a placeholder for testing.
```

Reply:
Thanks for the inputs. I have updated the files accordingly.